src/drawing/DrawingTarget.tsx

```diff
@@ -73,6 +78,9 @@ export const DrawingTarget = (props: Props, ref: React.Ref<DrawingTargetRef>): J
                 transform={`translate(-${centerX * 0.1}, -${centerY * 0.1}) scale(1.1)`}
                 {...shape}
             />
+            {pathGroups.map(pathGroup => (
+                <DrawingPathGroup key={uuid.v4()} isActive={isActive} pathGroup={pathGroup} rootEl={rootEl} />
```

Comment:
I agree with @ConradJChan that these keys should be stable between renders. We could use the item index if we're confident the order of the items won't change between renders. We could also assign a client-side id for each path group when they're stored in state by the reducer.